packages/core/core/src/requests/TargetRequest.js

```diff
@@ -91,6 +91,24 @@ async function run({input, api, options}: RunOpts) {
   );
   let targets = await targetResolver.resolve(input.packagePath, input.target);
 
+  for (const currTarget of targets) {
+    if (
+      currTarget.name !== 'default' &&
+      currTarget.distEntry != null &&
+      currTarget.distDir + '/' + currTarget.distEntry === input.filePath
```

Comment:
not sure if this'll help in resolving the windows test case issue but you probably want to use path utils for this, something like `path.join(currTarget.distDir, currTarget.distEntry) === input.filePath` or even `path.join(currTarget.distDir, currTarget.distEntry) === path.normalize(input.filePath)`

Reply:
@DeMoorJasper you're right. That code looks a lot cleaner. I'll use one of those if I'm able to debug this. Thank you.